Extensions/Effects/brightness-pixi-filter.ts

```diff
@@ -17,16 +17,32 @@ namespace gdjs {
         if (parameterName !== 'brightness') {
           return;
         }
-        brightnessFilter.brightness(
-          gdjs.PixiFiltersTools.clampValue(value, 0, 1),
-          false
-        );
+        const brightness = gdjs.PixiFiltersTools.clampValue(value, 0, 1);
+        //@ts-ignore
+        brightnessFilter.__brightness = brightness;
+        brightnessFilter.brightness(brightness, false);
+      }
+      getDoubleParameter(filter: PIXI.Filter, parameterName: string): number {
+        const brightnessFilter = (filter as unknown) as PIXI.ColorMatrixFilter;
+        if (parameterName === 'brightness') {
+          //@ts-ignore
+          return brightnessFilter.__brightness;
```

Comment:
Will it break if we change version for the pixi filter? Might need a comment about why you use this "private" attribute

Reply:
It won't break, but yes I should add a comment.